app/src/main/java/de/westnordost/streetcomplete/quests/bike_rental_capacity/AddBikeRentalCapacity.kt

```diff
@@ -0,0 +1,38 @@
+package de.westnordost.streetcomplete.quests.bike_rental_capacity
+
+import de.westnordost.streetcomplete.R
+import de.westnordost.streetcomplete.data.osm.mapdata.Element
+import de.westnordost.streetcomplete.data.osm.mapdata.MapDataWithGeometry
+import de.westnordost.streetcomplete.data.osm.mapdata.filter
+import de.westnordost.streetcomplete.data.osm.osmquests.OsmFilterQuestType
+import de.westnordost.streetcomplete.data.osm.osmquests.Tags
+import de.westnordost.streetcomplete.data.user.achievements.QuestTypeAchievement.BICYCLIST
+import de.westnordost.streetcomplete.osm.updateWithCheckDate
+
+class AddBikeRentalCapacity : OsmFilterQuestType<Int>() {
+
+    override val elementFilter = """
+        nodes, ways with
+         amenity = bicycle_rental
+         and access !~ private|no
+         and bicycle_rental ~ docking_station|dropoff_point
```

Comment:
I've enabled it for `bicycle_rental ~ docking_station|dropoff_point` (in #2129, only `docking_station` was suggested) as for both the capacity seems to be clearly visible.

Reply:
Dockless may well have the same counting issues I experience with bike hoops hence why I raised the issue, see e.g.:
http://prod-upp-image-read.ft.com/6cccaaac-76d3-11e7-90c0-90a9d1bc9691

Or the example node you linked to is here:
https://goo.gl/maps/fbYQxYoayNf4ihCm9

Part of the issue is the tagging scheme doesn't differentiate between dockless and a bike rack with reserved slots for a rental scheme (where the bike locks itself), lumping them in together. It only differentiates the docking station (where the rack locks the bike).

Reply:
You're right. I restricted the quest to `bicycle_rental=docking_station` in [`3773f50`](https://github.com/streetcomplete/StreetComplete/pull/4008/commits/3773f50cbf7865d76cb8fbcb32331c6f7243e469).

Reply:
If the tagging was ever fixed, we could always tweak it to cover the non-dockless ones too.